src/site/markdown/server/manage-webadmin.md

```diff
@@ -1151,6 +1152,34 @@ Response codes:
 
  - 200: List of users had successfully been returned.
  - 400: Validation issues with parameters
+ 
+### Recomputing current quotas for users
+
+This task is available on top of Guice products.
```

Comment:
WXebadmin is only available on top of guice product, this totology can be removed.

Reply:
Not available for memory product I bet